java/memory/memory-netty/src/main/java/io/netty/buffer/PooledByteBufAllocatorL.java

```diff
@@ -137,7 +137,6 @@ private class InnerAllocator extends PooledByteBufAllocator {
 
     private final PoolArena<ByteBuffer>[] directArenas;
     private final MemoryStatusThread statusThread;
-    private final int chunkSize;
```

Comment:
From code analysis, it seems that the `chunkSize` field here is useless. After the constructor is assigned, it is no longer used, and this value is directly derived from the parent class.